compose-recipes/sqlserver/liferay.sqlserver.yaml

```diff
@@ -0,0 +1,10 @@
+services:
+    liferay:
+        depends_on:
+            database:
+                condition: service_healthy
+        environment:
+            -   LIFERAY_JDBC_PERIOD_DEFAULT_PERIOD_DRIVER_UPPERCASEC_LASS_UPPERCASEN_AME=com.microsoft.sqlserver.jdbc.SQLServerDriver
+            -   LIFERAY_JDBC_PERIOD_DEFAULT_PERIOD_PASSWORD=Liferay123
```

Comment:
Is it possible to do no password to make connecting with a client simpler?

Reply:
unfortunately not; the mssql image requires the password to be set.